release-notes/opensearch-release-notes-1.3.1.md

```diff
@@ -0,0 +1,27 @@
+# OpenSearch 1.3.1 Release Notes
+
+## OpenSearch Security
+### Bug fixes
+
+* Fix for "java.lang.IllegalArgumentException: The array of keys must not be null" for "_cat/health" requests ([#1048](https://github.com/opendistro-for-elasticsearch/security/pull/1048))
+* Revert "Fix AuthCredentials equality (#876)" to improve performance ([#1061](https://github.com/opendistro-for-elasticsearch/security/pull/1061))
+
+### Maintenance
+
+* Bump version to 1.13.1.0 ([#1054](https://github.com/opendistro-for-elasticsearch/security/pull/1054))
```

Comment:
Why is there a couple of ODFE PRs in this?

Reply:
@cliu123 might be able to answer that

Reply:
That is security plugin ODFE 1.13 release note. The OpenSearch 1.3.1 release note is here: https://github.com/opensearch-project/security/blob/1.3/release-notes/opensearch-security.release-notes-1.3.1.0.md

Reply:
Thanks @cliu123. Fixed the release notes and format too.
cc: @peterzhuamazon